xml/System.CodeDom.Compiler/IndentedTextWriter.xml

```diff
@@ -1391,10 +1391,10 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">The buffer to write.</param>
+        <param name="cancellationToken">Token for canceling the operation.</param>
+        <summary>Asynchronously writes the specified buffer to the underlying <see cref="T:System.IO.TextWriter" />, inserting tabs at the start of every line.</summary>
```

Comment:
```suggestion
        <param name="buffer">The character array to write.</param>
        <param name="cancellationToken">Token for canceling the operation.</param>
        <summary>Asynchronously writes the character array to the underlying <see cref="T:System.IO.TextWriter" />, inserting tabs at the start of every line.</summary>
```

Reply:
I used 'characters' as it's not necessarily an array.